common/persistence/visibility/store/sql/query_converter.go

```diff
@@ -620,3 +620,13 @@ func isSupportedTypeRangeCond(saType enumspb.IndexedValueType) bool {
 	}
 	return false
 }
+
+func removeOrderByFromSelectQuery(queryString string) (string, error) {
+	stmt, err := sqlparser.Parse(queryString)
+	if err != nil {
+		return "", err
+	}
+	selectStmt, _ := stmt.(*sqlparser.Select)
+	selectStmt.OrderBy = nil
+	return sqlparser.String(selectStmt), nil
+}
```

Comment:
I don't like this, re-parsing just to remove the ORDER BY.
Can we modify the query converter to take a parameter to include default ORDER BY or not?

Reply:
Fair enough, let me update this.